tests/test_parameter_types.py

```diff
@@ -86,3 +86,24 @@ def test_standard_handling(self):
         standard_input = {'param': {'name': ['Sarah'], 'age': [25]}}
         result = standard_py_func(**standard_input)
         assert age == result
+
+
+class TestNestedType(object):
+
+    def test_nested_handling(self):
+        pd_data = {'name': ['Sarah'], 'age': [25]}
+        np_data = [('Sarah', (8.0, 7.0))]
+        std_data = {'name': ['Sarah'], 'age': [25]}
+        nested_input_data = {'input1': pd_data,
+                             'input2': np_data,
+                             'input3': std_data,
+                             'input0': 0}
+        result = nested_func(nested_input_data)
+        assert all(key in result.keys() for key in ('output1', 'output2', 'output3'))
```

Comment:
I know I've pointed out missing it in places, looks like output0 is missing here too, is there a reason for leaving it out?